src/lib/y2packager/release_notes_fetchers/url.rb

```diff
@@ -346,7 +346,8 @@ def curl_download(url, filename, max_time: 300)
           max_time
         )
         ret = Yast::SCR.Execute(Yast::Path.new(".target.bash"), cmd)
-        log.info("#{cmd} returned #{ret}")
+        filtered_cmd = cmd.sub(/--proxy-user '[^']*'/, "--proxy-user @PROXYPASSWORD@")
```

Comment:
You can use `curl_proxy_args` string directly (it is used few lines above), simply `cmd.sub(curl_proxy_args, "--proxy-user @PROXYPASSWORD@")`